packages/beacon-node/src/db/repositories/lightclientBestUpdate.ts

```diff
@@ -1,14 +1,39 @@
 import {IChainForkConfig} from "@lodestar/config";
 import {Bucket, IDatabaseController, Repository} from "@lodestar/db";
-import {altair, ssz, SyncPeriod} from "@lodestar/types";
+import {ssz, SyncPeriod, allForks} from "@lodestar/types";
+
+const SLOT_BYTE_COUNT = 8;
 
 /**
  * Best PartialLightClientUpdate in each SyncPeriod
  *
  * Used to prepare light client updates
  */
-export class BestLightClientUpdateRepository extends Repository<SyncPeriod, altair.LightClientUpdate> {
+export class BestLightClientUpdateRepository extends Repository<SyncPeriod, allForks.LightClientUpdate> {
   constructor(config: IChainForkConfig, db: IDatabaseController<Uint8Array, Uint8Array>) {
+    // Pick some type but won't be used
     super(config, db, Bucket.lightClient_bestLightClientUpdate, ssz.altair.LightClientUpdate);
```

Comment:
can we use new buckets here and below to make migration easier

Reply:
bump the bucket number for this key?